app.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+
+const { getTopics } = require("./controllers/topics.controllers")
+
+
+
+const app = express();
+
+app.use(express.json())
+
+app.get('/api', (req, res) => {
+    res.status(200).send({msg: 'Server is up and running.'})
+});
+
+app.get('/api/topics', getTopics);
+
```

Comment:
At the moment the 404 error seems to come from 'express'. Please take control and use app.all('*',()=>{}) to catch all wrong path. 

https://www.geeksforgeeks.org/express-js-app-all-function/